Dockerfile

```
@@ -9,19 +9,16 @@ FROM amazonlinux:2 as core_base
 #   * AWS CLI
 #   * Terraform
 
-ENV NODE_VERSION "16.x"
-ENV TERRAFORM_VERSION "1.5.3"
-ENV AWS_CLI_VERSION "2.13.25"
+ENV NODE_VERSION="20.x"
+ENV TERRAFORM_VERSION="1.9.2"
+ENV AWS_CLI_VERSION="2.17.13"
 
 # Add NodeJS and Yarn repos & update package index
-RUN \
-        yum install https://rpm.nodesource.com/pub_${NODE_VERSION}/nodistro/repo/nodesource-release-nodistro-1.noarch.rpm -y && \
-        yum install nodejs -y --setopt=nodesource-nodejs.module_hotfixes=1 && \
-        curl -sL https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo && \
-        yum update -y
+RUN curl -fsSL https://rpm.nodesource.com/setup_20.x | bash -
```

Should return this to the variable method:
```suggestion
RUN curl -fsSL https://rpm.nodesource.com/setup_${NODE_VERSION} | bash -
```

---

Shoot. Didn't mean to leave that there.

---

Good catch.